examples/react/base/src/ui/nav-link.tsx

```diff
@@ -0,0 +1,10 @@
+import { Link } from "wouter";
+
+export function NavLink() {
+  return (
+    <div style={{ display: "flex", gap: "8px" }}>
```

Comment:
Maybe we should think about how to set styling sometimes later.
Don't think people who use boiler-plate set inline style.